appknox/client.py

```diff
@@ -541,19 +561,20 @@ def set_endpoint(self, resource, resource_id=None):
     def get(self, **kwargs):
         resp = requests.get(
             self.endpoint, headers=self.headers, auth=self.auth,
-            params=kwargs
+            proxies=self.proxies, params=kwargs
```

Comment:
Here `resp = self.session.get(self.endpoint, headers=self.headers, auth=self.auth, params=kwargs)`. Since proxies are already set in requests session.

Reply:
done, set it for `get` `post` and `direct_get`